apps/core/scenes/passengerForm/BaggageBundles/BaggageBundle.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+
+import React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, RadioButton } from '@kiwicom/universal-components';
+import { Text } from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import BaggageTypeDetail from './BaggageTypeDetail';
+
+type Props = {|
+  +price: ?string,
+  +bags: any,
+  +bagIndex: number,
+  +onToggleCheck: any,
```

Comment:
Why any? I think It could be: 
```
+onToggleCheck: (string, number) => void
```

Reply:
Sorry about `any`! I forgot to change it for the actual types. I know it might be in other places as well 😞